src/collection/components/ShareCollectionModal.tsx

```diff
@@ -1,16 +1,140 @@
-import React, { FunctionComponent } from 'react';
+import { useMutation } from '@apollo/react-hooks';
+import { get } from 'lodash-es';
+import React, { Fragment, FunctionComponent, useState } from 'react';
 
-import { Modal, ModalBody } from '@viaa/avo2-components';
+import {
+	Button,
+	FormGroup,
+	Modal,
+	ModalBody,
+	RadioButton,
+	RadioButtonGroup,
+	Spacer,
+	Toolbar,
+	ToolbarItem,
+	ToolbarRight,
+} from '@viaa/avo2-components';
+import { Avo } from '@viaa/avo2-types';
+import toastService, { TOAST_TYPE } from '../../shared/services/toast-service';
+import { UPDATE_COLLECTION_PROPERTY } from '../collection.gql';
 
 interface ShareCollectionModalProps {
 	isOpen: boolean;
 	setIsOpen: (isOpen: boolean) => void;
+	isPublic: boolean;
+	updateCollectionProperty: (value: boolean, fieldName: string) => void;
+	collection: Avo.Collection.Response;
 }
 
+const shareOptions = [
+	{
+		name: 'private',
+		label: 'Niet openbaar',
+		isPublicized: false,
+	},
+	{
+		name: 'public',
+		label: 'Openbaar',
+		isPublicized: true,
```

Comment:
zouden we dit niet gewoon isPublic noemen? of is hier een reden voor?

Reply:
of `isPublished`

Reply:
ik zou zo dicht mogelijk blijven bij de database benaming:

![image](https://user-images.githubusercontent.com/1710840/63954526-8a389000-ca83-11e9-8de2-1ee26a6c2d76.png)